rgw/v2/lib/s3/lifecycle_ops.py

```diff
@@ -0,0 +1,49 @@
+import os, sys, glob
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../")))
+import v2.utils.log as log
+from v2.lib.resource_op import Config
+import datetime
+import json
+import v2.lib.resource_op as s3lib
+import v2.utils.utils as utils
+from v2.utils.utils import HttpResponseParser
+from v2.lib.exceptions import TestExecError
+import v2.lib.manage_data as manage_data
+from v2.lib.s3.write_io_info import IOInfoInitialize, BasicIOInfoStructure, BucketIoInfo, KeyIoInfo
+
+io_info_initialize = IOInfoInitialize()
+basic_io_structure = BasicIOInfoStructure()
+write_bucket_io_info = BucketIoInfo()
+write_key_io_info = KeyIoInfo()
```

Comment:
Can you remove these? these are not used in this file. Can you also remove unused imports also?

Reply:
Done. This file has been re-named to rgw/v2/lib/s3/lifecycle_validation.py and the unwanted imports have been removed from there.

Reply:
I still don't see this code being removed and unused imports are still present

Reply:
I have removed this file lifecycle_ops.py.